apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nux/src/post-published-modal/index.tsx

```diff
@@ -57,11 +60,18 @@ const PostPublishedModal: React.FC = () => {
 		setShouldShowFirstPostPublishedModal,
 	] );
 
-	const handleClick = ( event: React.MouseEvent ) => {
+	const handleViewPostClick = ( event: React.MouseEvent ) => {
 		event.preventDefault();
 		( window.top as Window ).location.href = link;
 	};
 
+	const handleNextStepsClick = ( event: React.MouseEvent ) => {
+		event.preventDefault();
+		(
+			window.top as Window
+		 ).location.href = `https://wordpress.com/setup/write/launchpad?siteSlug=${ siteUrl }`;
```

Comment:
Wondering how to build this URL in a way that will respect the current environment as opposed to hardcoding `wordpress.com`